db/migrate/20220810090655_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.blob :image
```

Comment:
- [ ] Great job on the overall project, but you need to keep in mind that the image you will be provided is a `link to an image` as mentioned in the project requirements, not a real image, so using string data type would make more sense. Kindly change this data type

Reply:
thanks, @zhadier for your answer. I change it